pkg/alertmanager/alertmanager.go

```diff
@@ -370,6 +373,13 @@ func (am *Alertmanager) mergePartialExternalState(part *clusterpb.Part) error {
 	return am.state.(*state).MergePartialState(part)
 }
 
+func (am *Alertmanager) getFullState() (*clusterpb.FullState, error) {
+	if state, ok := am.state.(*state); ok {
+		return state.GetFullState()
+	}
+	return nil, errors.New("ring-based replication not configured")
```

Comment:
Going to update this message to match https://github.com/cortexproject/cortex/commit/c34eb640df12bd6e9ff40dee057696184ad03dc8